lib/key_finder.rb

```diff
@@ -0,0 +1,73 @@
+require_relative "machine"
+require_relative "crackable"
+
+class Keyfinder < Machine
+  include Crackable
```

Comment:
This class is responsible for finding the key when given an encrypted message and date.